rfcs/text/0030-linux-event-model.md

```diff
@@ -1736,6 +2063,29 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
+### Ingestion mechanisms
+
+The Adaptive workload protections (AWP) sensor team, is working hard on integrating these new ECS fields into the endpoint agent codebase. The existing kprobe sensor tech will be upgraded to support much of the new widened process context defined in this RFC. At the same time eBPF sensor technology is being worked on and will eventually be the go to method for building these process events.
+
+### Usage mechanisms
+
+As outlined earlier in this document. The primary use for this data will be to drive the Session View (aka Terminal view, aka Sessionizer). Because of the hierarchical nature of this data, there are edge cases where some parts of the process tree are not captured, or have been filtered out (due to data volume concerns). This widenened process context (e.g entry_leader, session_leader, parent, group_leader) will allow the Session View to repair disjointed trees, and ensure it can represent the data as accurately as possible.
```

Comment:
```suggestion
As outlined earlier in this document, the primary use for this data will be to drive the Session View (aka Terminal view, aka Sessionizer). Because of the hierarchical nature of this data, there are edge cases where some parts of the process tree are not captured or have been filtered out (due to data volume concerns). This widened process context (e.g entry_leader, session_leader, parent, group_leader) will allow the Session View to repair disjointed trees and ensure it can represent the data as accurately as possible.
```

nit: Spelling, commas